code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -951,6 +951,9 @@
 			if(I)
 				jobpart = "[I.assignment]"
 
+	//AIs can't hear holograms
+	if(istype(speaker, /obj/effect/overlay/holo_pad_hologram))
```

Comment:
you should check if the hologram specifically belongs to the pad the AI is currently using, so they can hear other holograms still.